ts/defs.bzl

```diff
@@ -397,10 +399,26 @@ def ts_project(
             **common_kwargs
         )
 
+        # Users should build this target to get a failed build when typechecking fails
+        native.filegroup(
+            name = transitive_typecheck_target_name,
+            srcs = [name],
+            output_group = "transitive_typecheck",
+            **common_kwargs
+        )
+
         # Ensures the typecheck target gets built under `bazel test --build_tests_only`
         build_test(
             name = test_target_name,
             targets = [typecheck_target_name],
+            tags = ["manual"] + common_kwargs.get("tags", []),
```

Comment:
Wrong target for the manual tag, should be the `transitive_` one...

Reply:
That's what I get for trying to make a quick edit from my phone 🤦